src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/DefendantTwoEmailDTOGenerator.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.civil.notification.handlers;
+
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+
+public abstract class DefendantTwoEmailDTOGenerator extends EmailDTOGenerator {
+
+    protected DefendantTwoEmailDTOGenerator(NotificationsProperties notificationsProperties) {
```

Comment:
There should be no defendant two email generator as the lip v lip lip is not allowed. Claimant and defendant generator refers to lip v lip in our case.
If we are using lr v lip lip, go for appsolone, respsolone, respsoltwo